src/main/java/com/developerphil/adbidea/ui/MyDeviceChooser.java

```diff
@@ -112,7 +113,7 @@ public MyDeviceChooser(boolean multipleSelection,
 
     myFacet = facet;
     myFilter = filter;
-    myMinSdkVersion = AndroidModuleInfo.getInstance(facet).getRuntimeMinSdkVersion();
+    myMinSdkVersion = AndroidModuleModel.get(facet).getRuntimeMinSdkVersion();
```

Comment:
the `AndroidModuleInfo.getInstance(facet)` method is present in studio 3.0, which has just reached beta1 so it should be released soon enough. 

The current version of the plugin (1.4.1) has already dropped support for studio 2.3 ( well, users of 2.3 are still able to download the latest support that was compatible with studio2.3, which is adb idea 1.4.0). 

I don't plan to backport anything to 2.3 so we can safely revert this line.

Usually I try to support the current stable version and the current beta but this time I pushed a bug fix that broke this compatibility (I didn't think that any feature would be added before the 3.0 release). 

Tl;DR let's revert this line